transforms/generate-cleanup

```diff
@@ -109,5 +111,6 @@
 <transform dir file link hardlink -> \
 	edit target "\$!\((.*)\)" "$(\1)">
 
-<transform dir path=mangled -> drop>
-<transform dir file link hardlink path=mangled/(.*) -> drop>
+<transform file link hardlink path=mangled/(.*) -> drop>
+# drop all dirs except for illumos-closed
+<transform dir path=(?!(opt/onbld/closed/root_i386(-nd)?/)) -> drop>
```

Comment:
Can you verify this does not break sample-manifest targets for packages?

Reply:
I ran `gmake sample-manifest` for few packages.  I cannot run it for all packages, because it would take ages. OTOH, I do not think this could cause any problem.